packages/hub/src/index.ts

```diff
@@ -2,11 +2,11 @@
  * A web-gRPC wrapper client for communicating with the web-gRPC enabled Textile ThreadDB & Buckets APIs.
  * @packageDocumentation
  */
-export * from '@textile/users'
 export * from '@textile/buckets'
 export * from '@textile/crypto'
 export * from '@textile/grpc-authentication'
 export * from '@textile/hub-filecoin'
 export * from '@textile/hub-threads-client'
 export * from '@textile/security'
 export { ThreadID, Variant } from '@textile/threads-id'
+export * from '@textile/users'
```

Comment:
linter, is this "safe" @andrewxhill ?

Reply:
which part of it?

Reply:
that the linter moved export * from users to the bottom